source/textUtils.py

```diff
@@ -18,6 +17,8 @@
 from logHandler import log
 
 WCHAR_ENCODING = "utf_16_le"
+USERANSICODEPAGE = locale.getpreferredencoding()

```

Comment:
Please follow the style used for constants here:

```suggestion
USER_ANSI_CODE_PAGE = locale.getpreferredencoding()
```

Or rather

```suggestion
SYSTEM_PREFERRED_ENCODING = locale.getpreferredencoding()
```

Reply:
I've renamed it to `USER_ANSI_CODE_PAGE` because it can be set per  Windows user account and is not global to the system.